### DISCUSSION TITLE
大佬，为什么我一台可以用，另一台不可以

### DISCUSSION THREAD

**Discussion by rickShady1** [lisai9093/YYS#14] [Q&A] [answered]
17可以截图，其他指令都不行，截图显示1600 900是不是不对
![111](https://github.com/lisai9093/YYS/assets/156583424/594b0a1f-528d-480c-a84f-e4a4fa70e1fc)

**Comment by lisai9093** [comment, marked as answer]
分辨率必须是1136x640或者640x1136才行。脚本会默认尝试修改模拟器分辨率，如果不行的话得手动去MuMu模拟器修改。。。

**Comment by rickShady1** [comment]
非常感谢

**Reply by lincou** [reply]
我想问问mumu模拟器的版本是多少呢
我使用的时候检测不到mumu模拟器

**Reply by lisai9093** [reply]
> 我想问问mumu模拟器的版本是多少呢 我使用的时候检测不到mumu模拟器

可能是模拟器安装路径的问题，目前只支持默认安装位置："C:\\Program Files\\Netease\\MuMuPlayer-12.0\\shell\\adb.exe"
你可以手动把adb拷贝到这里，如果还有问题的话建议发个新帖子再讨论
